### PR TITLE
Fixed a variable name

### PR DESCRIPTION
**Pull request by hoddy3190**
+ Fixed dead links in README.md
+ Fixed an undefined variable `git_repo`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,9 +87,9 @@ If you want, you can initialize install manifest directory with [anyenv/anyenv-i
 If you have own manifest repository, you can specify it:
 
 ```
-$ anyenv install --init https://github.com/foo/anyenv-install.git
+$ anyenv install --init https://github.com/anyenv/anyenv-install.git
```
Comment (riywo) [inline README.md]: This is intentional since this option should be used by a custom manifests, not for default one.

File: libexec/anyenv-install [hunk 2]
```diff
@@ -125,7 +125,7 @@ initialize_manifest() {
     if [ -n "$force" ]; then
       fetch_git $ANYENV_DEFINITION_ROOT "$git_url" "$git_ref"
     else
-      read -p "Do you want to checkout ${git_repo}? [y/N]: " answer
+      read -p "Do you want to checkout ${git_url}? [y/N]: " answer
```
Comment (riywo) [inline libexec/anyenv-install:128]: Thanks, I can accept this change. Could you change the PR?

**Comment by hoddy3190** [comment]
I reverted "Fixed dead links" in https://github.com/anyenv/anyenv/pull/75/commits/4658738fee0b030b3685faa9e8a67de874ea7a81
Review again, please!

**Comment by riywo** [comment]
LGTM, but please squash commits and change PR title as well.

**Comment by hoddy3190** [comment]
> please squash commits and change PR title as well

I did it.

**Comment by n-rodriguez** [comment]
Hi there! Any news?

**Comment by riywo** [comment]
Thanks!

**Pull request merged**